src/style.css

```diff
@@ -1 +1,25 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
The `.stars` class is used for styling, which is good practice. However, ensure that if you use tag names for styling other elements, they are replaced with class names as per the checklist requirements.